### PR TITLE
Added Muted Topic screen in Setting Detail Screen.

### PR DESCRIPTION
**Pull request by jainkuniya**
screen shot 2017-08-04 at 4 21 08 pm
screen shot 2017-08-04 at 4 20 06 pm
screen shot 2017-08-04 at 4 20 23 pm
screen shot 2017-08-04 at 4 20 42 pm

### CODE REVIEW

**Comment by vishwesh3** [comment]
Updated.
Fixed travis build.

**Comment by vishwesh3** [comment]
rebased 👍

File: src/types.js [hunk 1]
```diff
@@ -356,6 +356,8 @@ export type ReactionType = {
 
 export type LocalizableText = string | { text: string, values: Object };
 
+export type MutedTopic = [];
```
Comment (kunall17) [inline src/types.js]: We can specify this at more detail 
`export type MutedTopic = [string, string]`

Comment (jainkuniya) [reply src/types.js]: Done 👍

**Comment by jainkuniya** [comment]
Rebased 👍 
ready to go in 🏃

**Comment by borisyankov** [comment]
Closing this, but feel free to move some of the work to the new topic screen.

**Pull request closed without merging**